code/controllers/subsystems/ticker.dm

```diff
@@ -0,0 +1,564 @@
+//
+// Ticker controls the state of the game, being responsible for round start, game mode, and round end.
+//
+SUBSYSTEM_DEF(ticker)
+	name = "Gameticker"
+	wait = 2 SECONDS
+	init_order = INIT_ORDER_TICKER
+	priority = FIRE_PRIORITY_TICKER
+	flags = SS_NO_TICK_CHECK | SS_KEEP_TIMING
+	runlevels = RUNLEVEL_LOBBY | RUNLEVEL_SETUP | RUNLEVEL_GAME | RUNLEVEL_POSTGAME // Every runlevel!
+
+	var/const/restart_timeout = 3 MINUTES	// Default time to wait before rebooting in desiseconds.
+	var/current_state = GAME_STATE_INIT	// We aren't even at pregame yet // TODO replace with CURRENT_GAME_STATE
+
+	/* Relies upon the following globals (TODO move those in here) */
+	// var/master_mode = "extended"		//The underlying game mode (so "secret" or the voted mode).
+										// Set by SSvote when VOTE_GAMEMODE finishes.
+	// var/round_progressing = 1		//Whether the lobby clock is ticking down.
+
+	var/pregame_timeleft = 0			// Time remaining until game starts in seconds. Set by config
+	var/start_immediately = FALSE		// If true there is no lobby phase, the game starts immediately.
+
+	var/hide_mode = FALSE 				// If the true game mode should be hidden (because we chose "secret")
+	var/datum/game_mode/mode = null		// The actual gamemode, if selected.
+
+	var/end_game_state = END_GAME_NOT_OVER	// Track where we are ending game/round
+	var/restart_timeleft				// Time remaining until restart in desiseconds
+	var/last_restart_notify				// world.time of last restart warning.
+	var/delay_end = FALSE               // If set, the round will not restart on its own.
+
+	var/login_music						// music played in pregame lobby
+
+	var/list/datum/mind/minds = list()	// The people in the game. Used for objective tracking.
+
+	// TODO - I am sure there is a better place these can go.

```

Comment:
GLOB? Or perhaps the job controller?

Reply:
Either would be good.  In fact there are several vars throughout the system which ought to be re-oraganized onto GLOB or the job controller.  But I think that is best left to those PRs so this is focused on SSticker, leaving the vars that were on ticker before to be on ticker still.